docs/source/index.rst

```diff
@@ -41,16 +41,21 @@ Setting `return_indices_ranked_by` in this function instructs cleanlab to return
 
 .. code-block:: python
 
-   from cleanlab.filter import find_label_issues
-
-   ordered_label_issues = find_label_issues(
-       labels=labels,
-       pred_probs=pred_probs,
-       return_indices_ranked_by='self_confidence',
-   )
+    from cleanlab.classification import CleanLearning
+    from cleanlab.filter import find_label_issues
+    
+    # Option 1 - sklearn-compatible models - just provide the data and labels ツ
+    issues_df = CleanLearning(clf=sklearnCompatibleModel).find_label_issues(data, labels)
```

Comment:
what about either: moving this CleanLearning portion into to its own block, or explaining it a bit more in the text above?  Currently the text above is all about `filter.find_label_issues` but then this code block jumps into CleanLearning first before presenting the `filter` code that was discussed in the text above

Reply:
I actually think there is 'too much' text above it. I think these examples are strong stand-alone, as in this stackoverflow post: https://stackoverflow.com/questions/40009134/dealing-with-noisy-training-labels-in-text-classification-using-deep-learning/53604373#53604373

Reply:
@jwmueller i can probably get the text above down to 50%, but it will take an hour to do well and want to make sure thats cool w/ you before i spend the time.

Reply:
ya that sounds worthwhile, especially to avoid the confusion that new CleanLearning is being presented first. I agree the text above can be minimal